packages/react-components/react-aria/src/activedescendant/scrollIntoView.ts

```diff
@@ -1,8 +1,11 @@
-export const scrollIntoView = (target: HTMLElement | null | undefined) => {
+export const scrollIntoView = (target: HTMLElement | null | undefined, winArg?: Window | null) => {
   if (!target) {
     return;
   }
 
+  // eslint-disable-next-line no-restricted-globals
+  const win = winArg ?? window;
```

Comment:
IMO we should do there `const win = target.ownerDocument.defaultView` and do not fallback do a default `window` at all